server/ngwmn/views.py

```diff
@@ -9,7 +9,6 @@
 
 from . import __version__, app
 from .services.ngwmn import get_features, get_water_quality, get_well_log, get_statistics, get_providers, get_sites
-from .services.sifta import get_cooperators
```

Comment:
I removed the call to sifta but left in the code in services.sifta since the intention is to use sifta in the near future.